### PR TITLE
VACMS-12994: Install node and npm on DDEV and Tugboat

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Relates to #12994 

## Testing done
Tested on base preview https://tugboat.vfs.va.gov/641b1f2a63b3e8775e1133a9

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: .tugboat/config.yml [hunk 1]
```diff
@@ -128,6 +128,10 @@ services:
         - pecl install memcache-8.0
         - docker-php-ext-enable memcache
 
+        # Install node and npm.
+        - curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+        - apt-get install -y nodejs
+
```
Comment (ndouglas) [inline .tugboat/config.yml:134]: Why do we still have these instructions if we have the `install-nvm.sh` script below?

I really feel like you're on the right track with the `install-nvm.sh` script -- there's just some tinkering to be done to get `nvm` to be discoverable.

Comment (ndouglas) [reply .tugboat/config.yml:134]: Oh, I remember you telling me now -- DDEV vs. Tugboat.

Comment (edmund-dunn) [reply .tugboat/config.yml:134]: install-nvm.sh is being used by ddev. Since this is a blocker for #12930 I just need to get it done.

**Comment by ndouglas** [comment]
Needs a Composer Validate, then we can merge without rerunning the tests.

**Review by ndouglas — APPROVED** [review]
YES

**Pull request merged**